src/Form/Mapper/FormMapper.php

```diff
@@ -21,56 +21,30 @@
 interface FormMapper
 {
     /**
-     * @param string $name
-     * @param mixed  $type
-     *
-     * @return FormBuilderInterface
+     * @param mixed $type
      */
-    public function create($name, $type = null, array $options = []);
+    public function create(string $name, ?string $type = null, array $options = []): FormBuilderInterface;
```

Comment:
Except of what I suggested before about the 2nd nullable argument, the other changes are fine on my side.

Reply:
Sorry, aren't you happy with the justification I gave?

Reply:
Didn't see the last comments, sorry. I'm fine with that then. I'm pretty dizzy with the LSP implementation changes ([covariants](https://wiki.php.net/rfc/covariant-returns-and-contravariant-parameters) and co, etc). I have a vague memory about this syntax being discouraged when the argument already defines `null` as default value, but I can't confirm that now.